MANUAL.md

```diff
@@ -2253,8 +2253,65 @@ uses [custom options](#custom-options).
 
 ## Migrating from version 1
 
```

Comment:
Add in the 2 lines from the blog post upgrade section regarding Connect-ES and not needing to upgrade if they don't need Editions: https://github.com/bufbuild/buf.build/pull/1813#discussion_r1715575450

Reply:
I don't think we should put the contemporary information in the manual.

Reply:
Yeah, I think I agree...we have a lot of commentary on the blog post :)